esp/src/src/ECLArchiveWidget.ts

```diff
@@ -121,13 +137,44 @@ export class ECLArchiveWidget {
                     ;
 
                 wu.fetchArchive().then(function (archiveXML) {
-                    renderArchive(archiveXML);
+                    wu.fetchDetailsRaw({
+                        ScopeFilter: {
+                            MaxDepth: 999999,
+                            ScopeTypes: ["graph"]
+                        },
+                        ScopeOptions: {
+                            IncludeMatchedScopesInResults: true,
+                            IncludeScope: true,
+                            IncludeId: true,
+                            IncludeScopeType: true
+                        },
+                        PropertyOptions: {
+                            IncludeName: true,
+                            IncludeRawValue: true,
+                            IncludeFormatted: true,
+                            IncludeMeasure: true,
+                            IncludeCreator: true,
+                            IncludeCreatorType: true
+                        },
+                        NestedFilter: {
+                            Depth: 999999,
+                            ScopeTypes: ["activity"]
+                        },
+                        PropertiesToReturn: {
+                            AllStatistics: true,
+                            AllAttributes: true,
+                            AllHints: true,
+                            AllProperties: true,
+                            AllScopes: true
+                        }
+                    }).then(scopes=>{
+                        const markerData = buildMarkerData(scopes);
+                        renderArchive(archiveXML, markerData);
+                    });
                 });
```

Comment:
It might be worth while making these two calls in parallel?:
```
Promise.all([wu.fetchArchive(), wu.fetchDetailsRaw({...})]).then(([archiveXML, scopes])=>{});
```